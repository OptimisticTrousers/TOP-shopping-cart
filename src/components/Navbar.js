export default function Navbar() {
    return (
        <nav>Stuff</nav>
    )
}