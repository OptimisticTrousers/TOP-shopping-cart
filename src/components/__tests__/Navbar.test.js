import React from 'react'
import {render, screen} from '@testing-library/react'
import '@testing-library/jest-dom'
import userEvent from '@testing-library/user-event'
import Navbar from '../Navbar'

it("renders navbar", () => {

    const {asFragment} = render(<Navbar />)

    expect(asFragment()).toMatchSnapshot()
})