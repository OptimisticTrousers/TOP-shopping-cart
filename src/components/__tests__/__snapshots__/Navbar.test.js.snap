// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders navbar 1`] = `
<DocumentFragment>
  <nav>
    Stuff
  </nav>
</DocumentFragment>
`;
